services/maven-central/maven-central.tester.js

```diff
@@ -22,18 +22,8 @@ t.create('latest 0.8 version')
 
```

Comment:
One last ask from me (sorry I missed this earlier) 😄 

Let's switch to the `createServiceTester` shorthand:
`const t = (module.exports = require('../tester').createServiceTester())`

And I'd also suggest we reuse one of existing test validator for versions (they're used for other maven version related badges)
`isVPlusDottedVersionAtLeastOne` or `isVPlusDottedVersionNClausesWithOptionalSuffix` from `test-validators` should work here too

Reply:
I'll probably go with `isVPlusDottedVersionNClausesWithOptionalSuffix`.
Might wanna add some nock tests for that.